File: src/main/java/com/wgzhao/addax/admin/model/AddaxLog.java
package com.wgzhao.addax.admin.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Entity
@Table(name = "addax_log")
@Data
@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class AddaxLog
{
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private Long tid;
    private LocalDateTime runAt;
    private LocalDate runDate;
    private String log;
}

File: src/main/kotlin/com/wgzhao/addax/admin/model/AddaxLog.kt
package com.wgzhao.addax.admin.model

import jakarta.persistence.*
import java.time.LocalDate
import java.time.LocalDateTime

@Entity
@Table(name = "addax_log")
data class AddaxLog(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,
    var tid: Long? = null,
    var runAt: LocalDateTime,
    var runDate: LocalDate,
    var log: String? = null
)
